### PR TITLE
extends filters to all equipment types

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java [hunk 1]
```diff
@@ -315,4 +433,98 @@ private StringBuilder numericalRange(String id, RangeType range, Double value1,
...
+                .andExpect(status().isOk())
+                .andReturn().getResponse().getContentAsString(),
+            new TypeReference<>() {
+            });
+
+        assertEquals(1, filterAttributes.size());
+        assertEquals(name, filterAttributes.get(0).getName());
+        assertEquals(id, filterAttributes.get(0).getId());
+        assertEquals(type, filterAttributes.get(0).getType());
+        assertEquals(description, filterAttributes.get(0).getDescription());
```
Comment (niconoir) [inline src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java:478]: Why not check nominalVoltage and countries ?

File: src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java [hunk 2]
```diff
@@ -315,4 +433,98 @@ private StringBuilder numericalRange(String id, RangeType range, Double value1,
...
```
Comment (niconoir) [inline src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java:526]: Why not check nominalVoltage and countries ?

File: src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java:478]: It is already tested in the insertFilter method called from this method

File: src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java:526]: It is already tested in the insertFilter method called from this method

File: src/main/java/org/gridsuite/filter/server/ThreeWindingsTransformerFilterRepositoryProxy.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    @Override
+    public ThreeWindingsTransformerFilterEntity fromDto(AbstractFilter dto) {
+        if (dto instanceof ThreeWindingsTransformerFilter) {
+            var threeWindingsTransformerFilter = (ThreeWindingsTransformerFilter) dto;
+            var threeWindingsTransformerFilterEntityBuilder = ThreeWindingsTransformerFilterEntity.builder()
+                .equipmentName(threeWindingsTransformerFilter.getEquipmentName())
+                .equipmentId(threeWindingsTransformerFilter.getEquipmentID())
+                .substationName(threeWindingsTransformerFilter.getSubstationName())
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/filter/server/ThreeWindingsTransformerFilterRepositoryProxy.java:56]: That's right. I removed these 2 lines here ...

File: src/main/java/org/gridsuite/filter/server/TwoWindingsTransformerFilterRepositoryProxy.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            entity).build();
+    }
+
+    @Override
+    public TwoWindingsTransformerFilterEntity fromDto(AbstractFilter dto) {
+        if (dto instanceof TwoWindingsTransformerFilter) {
+            var twoWindingsTransformerFilter = (TwoWindingsTransformerFilter) dto;
+            var twoWindingsTransformerFilterEntityBuilder = TwoWindingsTransformerFilterEntity.builder()
+                .equipmentName(twoWindingsTransformerFilter.getEquipmentName())
+                .equipmentId(twoWindingsTransformerFilter.getEquipmentID())
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/filter/server/TwoWindingsTransformerFilterRepositoryProxy.java]: That's right. I removed these 2 lines here ...

**Review by niconoir — APPROVED** [review]

**Pull request merged**